Occasional `docker buildx imagetools` error pushing images to dockerhub in CI

This is happening with a reasonable frequency

It seems like the issue is probably related to the skopeo version

```console
github.com/containerd/containerd/images.HandlerFunc.Handle(0x0?, {0x237da00?, 0xc000803800?}, {{0xc0006790b0, 0x2b}, {0xc000675360, 0x47}, 0x318, {0x0, 0x0, ...}, ...})
	/build/moby-buildx/.gopath/src/github.com/docker/buildx/vendor/github.com/containerd/containerd/images/handlers.go:59 +0x5f
github.com/containerd/containerd/images.Dispatch.func1()
	/build/moby-buildx/.gopath/src/github.com/docker/buildx/vendor/github.com/containerd/containerd/images/handlers.go:168 +0xd6
golang.org/x/sync/errgroup.(*Group).Go.func1()
	/build/moby-buildx/.gopath/src/github.com/docker/buildx/vendor/golang.org/x/sync/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
	/build/moby-buildx/.gopath/src/github.com/docker/buildx/vendor/golang.org/x/sync/errgroup/errgroup.go:72 +0xa5
##[error]Bash exited with code '2'.
```

--- a/ci/docker_ci.sh
+++ b/ci/docker_ci.sh
@@ -279,7 +279,11 @@ tag_image () {
 
     if [[ -z "$DOCKER_CI_DRYRUN" ]]; then
         echo "..."
-        docker "${docker_tag_args[@]}"
+        docker "${docker_tag_args[@]}" || {
+            echo "Retry Docker tag in 5s ..." >&2
+            sleep 5
+            docker "${docker_tag_args[@]}"
+        }
     fi
 }
 

